src/ChartType.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Atk4\Chart;
+
+/**
+ * Supported chart types.
+ */
+class ChartType
```

Comment:
no class needed, see comment above

Reply:
Can do like that, but is that better? I moved them out to slightly make Chart class code smaller, but if you think it's better to be moved into Chart class, then fine with me.

Reply:
IMHO it is better and consistent with other atk4 "enum"s

Reply:
done